dash/testing/wait.py

```diff
@@ -92,8 +100,7 @@ def __call__(self, driver):
                 or str(elem.get_attribute("value")) == self.text
             )
         except WebDriverException:
-            logger.exception("text_to_equal encountered an exception")
```

Comment:
We could get rid of the whole `try/except` here instead, and just let `WebDriverException` be raised.